direct-auth-samples/src/test/java/pages/RootPage.java

```diff
@@ -60,6 +60,11 @@ public void navigateToTheRootPage() {
         driver.get("http://localhost:8080"); // TODO pass as env variable.
     }
 
+    public void navigateToThePasswordResetPage() {
+        driver.manage().window().maximize();
+        driver.get("http://localhost:8080/forgot-password"); // TODO pass as env variable.
```

Comment:
Please create a followup/backlog JIRA to address this.

Reply:
Int ref:  [OKTA-401435](https://oktainc.atlassian.net/browse/OKTA-401435)